pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -367,7 +374,7 @@ def apply(self):
         if self.autopart:
             self.clearPartType = CLEARPART_TYPE_ALL
         else:
-            self.clearPartType = CLEARPART_TYPE_NONE
+            self.clearPartType = None
```

Comment:
Why this change? What the `None` here means?

Reply:
This change is a leftover from an older version. But you know what, the ``self.clearPartType`` attribute is only set in this class and never used. So I'm just removing it instead.